extensions/librdkafka/ConsumeKafka.cpp

```diff
@@ -78,7 +78,8 @@ core::Property ConsumeKafka::TopicNames(core::PropertyBuilder::createProperty("T
   ->build());
 
 core::Property ConsumeKafka::TopicNameFormat(core::PropertyBuilder::createProperty("Topic Name Format")
-  ->withDescription("Specifies whether the Topic(s) provided are a comma separated list of names or a single regular expression.")
+  ->withDescription("Specifies whether the Topic(s) provided are a comma separated list of names or a single regular expression. "
+      "Using regular expressions does not automatically discover kafka topics created after the processor started.")
```

Comment:
`Kafka` is written with a capital `K` elsewhere, so should probably be here, too

Reply:
Update in cdb014f306f614bfad5e6120f089ab821c904677